dll/win32/urlmon/CMakeLists.txt

```diff
@@ -3,8 +3,7 @@ add_definitions(
     -D_URLMON_
     -DENTRY_PREFIX=URLMON_
     -DPROXY_DELEGATION
-    -DWINE_REGISTER_DLL
-    -DPROXY_CLSID_IS={0x79EAC9F1,0xBAF9,0x11CE,{0x8C,0x82,0x00,0xAA,0x00,0x4B,0xA9,0x0B}})
```

Comment:
> > What could be the triggering difference to ROS and GitHub builders (which do not have this error)?
> 
> ~Maybe RosBE-Unix version? The latest release is 2.2.1 while builders use the unreleased 2.3.~ Yet the ROS hosted builders seem to use 2.2...

GH builders use 2.2.1 and do not report this error.
Then, we still don't know why your system triggers it.

> It seems to treat each comma separated value as another "definition", that's why it prints 10 redefinition errors for each build object.

Good clue!